src/app/page.tsx

```diff
@@ -0,0 +1,5 @@
+import HomePage from "./home-page";
```

Comment:
I think it's good to avoid having 2 different files named page for the same route. 

We should only keep the parts that need to be client components as client components and try and do as much as possible on the server side for speed.

Reply:
The layout can be done in the parent page. Just keep the parts needed for the client over there and rename to `home-components` to avoid confusion.